tests/unit/test_flight.py

```diff
@@ -1108,6 +1111,19 @@ def test_resample_and_fill_short_flight() -> None:
     fl3 = fl.resample_and_fill("1 min", keep_original_index=True)
     assert len(fl3) == 2
 
+    fl = Flight(
```

Comment:
Can you add an example of the other case, e.g.

```
...
time=[np.datetime64("2022-01-01T00:01:03"), np.datetime64("2022-01-01T00:01:59")],
```

and assert that the length is 0?

Reply:
There's an example of the other case earlier in this test (not visible in the diff because it was already there!)